components/automate-ui/src/app/app.module.ts

```diff
@@ -89,15 +89,15 @@ import { ApplicationsComponent } from './pages/applications/applications.compone
 import { AttributesComponent } from './page-components/attributes/attributes.component';
 import { AutomateSettingsComponent } from './pages/automate-settings/automate-settings.component';
 import { ClientRunsComponent } from './pages/client-runs/client-runs.component';
+import {
+  SearchBarComponent
+} from './page-components/search-bar/search-bar.component';
 import {
   ClientRunsSidebarComponent
 } from './page-components/client-runs-sidebar/client-runs-sidebar.component';
 import {
-  ClientRunsSearchBarComponent
-} from './page-components/client-runs-search-bar/client-runs-search-bar.component';
-import {
-  ClientRunsSearchFiltersComponent
-} from './page-components/client-runs-search-filters/client-runs-search-filters.component';
+  SearchBarFilterBarComponent
+} from './page-components/search-bar-filter-bar/search-bar-filter-bar.component';
```

Comment:
This list of imports is/was alphabetized. At one point there was a linter to enforce this but I'm not sure if that's still the case and I haven't checked if you're getting lint failures on this.

Reply:
I am not seeing any failures in buildkite. I updated the imports to have them alphabetized. 

Is this the only file that has its imports alphabetized?

Reply:
If I remember right, this was something we tried to setup but it didn't work for whatever reason. I think that one list was kept alphabetical just out of convention, since it was so long it was nice to keep it easily scannable.